### PR TITLE
CCDI Init

### PR DESCRIPTION
**Pull request by jordan-rash**

### CODE REVIEW

**Comment by jordan-rash** [comment]
CC: @adthrasher @claymcleod 
This repo won't let me add reviewers...so this is my CC hack

File: README.md [hunk 1]
```diff
@@ -1,11 +1,55 @@
...
+   <a href="https://github.com/stjude/teltale/actions?query=workflow%3APackage" target="_blank">
+     <img alt="Actions: Package Status"
+          src="https://github.com/stjude/teltale/workflows/Package/badge.svg" />
+   </a>
+  </p>
+
+
+  <p align="center">
+  
+`teltale` is a program that computes the fraction of telomeric reads in a BAM file. A read is considered telomeric if it contains at least seven consecutive occurrences of the telomere motif TTAGGG or its reverse complement CCCTAA. The fraction of telomeric reads is the number of telomeric reads divided by the total number of reads in the BAM file. Reads marked as Duplicate or Failed QC are ignored and are excluded from the counts. This program is intended for BAM files containing whole-genome sequencing (WGS) data.
```
Comment (adthrasher) [inline README.md:38]:
```suggestion
`teltale` is a program that computes the fraction of telomeric reads in a BAM file. A read is considered telomeric if it contains at least seven consecutive occurrences of the telomere motif `TTAGGG` or its reverse complement `CCCTAA`. The fraction of telomeric reads is the number of telomeric reads divided by the total number of reads in the BAM file. Reads marked as Duplicate or Failed QC are ignored and are excluded from the counts. This program is intended for BAM files containing whole-genome sequencing (WGS) data.
```

File: cgc/teltale.cwl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+$namespaces:
+  sbg: 'https://sevenbridges.com'
+baseCommand: []
+inputs:
+  - id: input_file
+    type: File
+    inputBinding:
+      position: 1
+    label: Input BAM file
+    'sbg:fileTypes': BAM
```
Comment (adthrasher) [inline cgc/teltale.cwl:12]: Is there a CWL generic option to do this also? Something that would enforce a ".bam" extension.

**Review by adthrasher — APPROVED** [review]

File: cgc/teltale.cwl (hunk 2 above)
Comment (jordan-rash) [reply cgc/teltale.cwl:12]: Not that I have seen, but I really haven't been looking for anything that does enforcement....that `sbg:fileTypes` just allows the UI to do some things as far as I can tell.  But I can put .bam at the end of any file and CGC will treat it as such

Comment (adthrasher) [reply cgc/teltale.cwl:12]: I was thinking of non-CGC use cases, but let's leave it as is.

Comment (jordan-rash) [reply cgc/teltale.cwl:12]: Either way, Ill investigate.

**Pull request merged**